### DISCUSSION TITLE
Error de paquete de analisis

### DISCUSSION THREAD

**Discussion by Osalex890** [skyline-emu/skyline#1593] [Q&A]
Por que no me deja intalar skyline me aparece 
Error de paquete de analisis

**Comment by DanXG790** [comment]
Because you have Android 9 or earlier, Skyline only works from Android 10 onwards

**Reply by Crytonics** [reply]
Nah i think he want to update skyline from 1200 to 1202. But 1202 is from master branch and 1200 is from ftx1 branch. Master branch dosen't have updates from ftx1 so games won't work

**Comment by Crytonics** [comment]
Just stay on 1200 build, because that is lastest build from ftx1 branch for now.
